store/blocks/common.json

```diff
@@ -18,7 +18,7 @@
     }
   },
   "flex-layout.row#product-name": {
-    "children": ["product-name"]
+    "children": ["vtex.product-list:product-name"]
```

Comment:
How about adding a comment to explain why is this useful?

Also, why not prefix all other blocks from this block used in this default implementation? We might even adopt this as a good practice for other apps that define a default implementation.

Reply:
I'm just solving a conflict with store-components. I don't think that this should be a good practice, we should solve this problem with something else because this is too verbose.